src/aditiGJUN20/ExtractMiddleChar.java

```diff
@@ -0,0 +1,32 @@
+/*
+Create a new method to print middle character of all words.
+
+Input = "Harsh Maulik";
+Output : r is middle character in Harsh
+              u is middle character in Maulik 
+ 
+*/
+
+package aditiGJUN20;
+
+public class ExtractMiddleChar {
+	
+	// Method to find middle character of String.
+	void getMidCharOfString(String[] array) {
+		for(int i=0;i<array.length;i++) {
+			String name=array[i];
+			if(name.length()%2 ==0)
+				System.out.println(String.valueOf(name.charAt((name.length()/2)-1))+" is middle character in "+name);
```

Comment:
This will surely work.. but what about below code.
For even or odd length string.
(name.length()-1)/2. This would be without condition.